spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaSettings.java

```diff
@@ -150,6 +155,93 @@ public Port setProtocol(SessionProtocol protocol) {
         }
     }
 
+    /**
+     * Configurations for the HTTP content encoding.
+     */
+    public static class Compression {
```

Comment:
Picked `Compression` because this is similar property to the Spring's `Compression` class. Please let me know if you think another name is proper. :-)